spmd/tensor/ops/math_ops.py

```diff
@@ -45,3 +45,23 @@ def softmax_bwd_rule(op_schema: OpSchema) -> OutputSharding:
             "Cannot run _softmax_backward_data on sharding dimension!"
         )
     return pointwise_rule(op_schema)
+
+
+@register_prop_rule("aten.native_dropout.default")
```

Comment:
Isn't dropout a pointwise rule? Are we specialcasing because of the non-deterministic behavior?

Reply:
yeah we are specializing because of non-deterministic behavior, but it is indeed a pointwise rule, let me move it to pointwise file instead and add a comment.